docs/index.md

```diff
@@ -139,7 +139,7 @@ Roles
 
 pulp_installer is equipped with the following roles:
 
-- [pulp](roles/pulp/): installs Pulp 3 from PyPi or source and provides basic config.
+- [pulp_common](roles/pulp_common): installs Pulp 3 from PyPi or source and provides basic config.
 - [pulp_content](roles/pulp_content): install, configure, and set the state of pulp content app.
```

Comment:
I think we can add `pulp_api` here

Reply:
It's not showing this as outdated, but I fixed it.